projects/storefrontlib/src/layout/a11y/skip-link/service/skip-link.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { SkipLinkConfig, SkipLinkScrollPosition } from '../config/index';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SkipLinkService {
+  skippers = new BehaviorSubject([]);
```

Comment:
I like to make this one private, and introduce a public method to expose the links, say `getSkipLinks`. With that, we can change the internal structure over time. The method should return a more generic observable rather than the subject. 

I also don't know the type of the _skippers_ array items. Is it an `HTMLElement`? 

Also, let's stick to similar terminology (I think SkipLink is the term to be used).